client/blocks/import/capture/test/index.tsx

```diff
@@ -85,4 +85,149 @@ describe( 'CaptureInput', () => {
 
 		expect( screen.getByRole( 'button', { name: /XYZ/ } ) ).toBeInTheDocument();
 	} );
+
+	it( 'should show no TLD error', async () => {
+		const onInputEnter = jest.fn();
+		const { getByText } = render(
+			<MemoryRouter>
+				<CaptureInput onInputEnter={ onInputEnter } />
+			</MemoryRouter>
+		);
+
+		await userEvent.type( screen.getByLabelText( /Enter the URL of the site/ ), 'myblog' );
+
+		await userEvent.click( screen.getByRole( 'button', { name: /Continue/ } ) );
+
+		expect(
+			getByText(
+				'Your URL is missing a top-level domain (e.g., .com, .net, etc.). Example URL: example.com'
+			)
+		).toBeInTheDocument();
+	} );
+
+	it( 'should show no TLD error for URL with protocol', async () => {
+		const onInputEnter = jest.fn();
+		const { getByText } = render(
+			<MemoryRouter>
+				<CaptureInput onInputEnter={ onInputEnter } />
+			</MemoryRouter>
+		);
+
+		await userEvent.type( screen.getByLabelText( /Enter the URL of the site/ ), 'https://myblog' );
+
+		await userEvent.click( screen.getByRole( 'button', { name: /Continue/ } ) );
+
+		expect(
+			getByText(
+				'Your URL is missing a top-level domain (e.g., .com, .net, etc.). Example URL: example.com'
+			)
+		).toBeInTheDocument();
+	} );
+} );
+
+// Helper function to enter URL and click CTA.
+const enterUrlAndContinue = async ( url ) => {
+	await userEvent.type( screen.getByLabelText( /Enter the URL of the site/ ), url );
+	await userEvent.click( screen.getByRole( 'button', { name: /Continue/ } ) );
+};
+
+describe( 'URL Validation', () => {
+	beforeEach( () => {
+		const onInputEnter = jest.fn();
+		render(
+			<MemoryRouter>
+				<CaptureInput onInputEnter={ onInputEnter } />
+			</MemoryRouter>
+		);
+	} );
+
+	test( 'should show error for missing top-level domain', async () => {
```

Comment:
Usually, we use the `it` instead of `test` the file already has scenarios with `it`. 

```suggestion
	it.( 'shows error for missing top-level domain', async () => {
```

Reply:
Done under it.each here https://github.com/Automattic/wp-calypso/pull/91748/commits/3447cd32549e85efbeeba51ca3f8ec7882a64e3d